python/lsst/ap/association/utils.py

```diff
@@ -89,7 +91,7 @@ def readSdmSchemaFile(schemaFile: str,
 
     Returns
     -------
-    schemaTable : 'dict' of `lsst.dax.apdb.apdbSchema.ApdbSchema`
+    schemaTable : 'dict' of `lsst.dax.apdb.schema_model.Table`
```

Comment:
```suggestion
    schemaTable : `dict` [`str`, `lsst.dax.apdb.schema_model.Table`]
```
Same for all the other places that say "'dict' of" instead of "`dict` of"